hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/TestFsDatasetImpl.java

```diff
@@ -201,9 +203,9 @@ public void setUp() throws IOException {
     assertEquals(0, dataset.getNumFailedVolumes());
   }
 
-  @Test
+  @Test(timeout=10000)
```

Comment:
If this times out, are we cleaning up correctly?

Reply:
`testReadLockEnabledByDefault` does not start a cluster. It has a mock datanode.
For `testReadLockCanBeDisabledByConfig` the cluster shutdown is inside `finally` block